source/sass/style.scss

```
@@ -1,12 +1,34 @@
+//viewports
+$tablet-width: 768px;
+$desctop-width: 1200px;
```

```suggestion
$desktop-width: 1200px;
```
(Нужно будет исправить везде, где используется эта переменная.